packages/ui-components/src/util/toAddress.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import keyring from '@polkadot/ui-keyring';
+import { hexToU8a, isHex } from '@polkadot/util';
+
+export default function toAddress (value?: string | Uint8Array): string | undefined {
```

Comment:
Can you add a FIXME here?

I think all UI code that are copied from polkadot/apps could be factorized and should go in https://github.com/polkadot-js/ui